Source/Internal/API/APIEndpoints.swift

```diff
@@ -57,7 +58,7 @@ internal enum APIEndpoints {
 
         case .virtusize(let region):
             components.host = "static.api.virtusize.\(region.rawValue)"
-            components.path = "/a/aoyama/latest/sdk-webview.html"
+            components.path = "/a/aoyama/testing/auth-sdk-events/sdk-webview.html"
```

Comment:
@akueisara Should we have `testing` here?
If so don't forget to change it to `latest` when released

Reply:
@zr0z Yes, I have a unit test to verify this. Right now the test is not passed and that will remind me of changing it when released.